lib/grandpa/grandpa.go

```diff
@@ -27,7 +27,8 @@ const (
 )
 
 var (
-	logger = log.NewFromGlobal(log.AddContext("pkg", "grandpa"))
+	logger                 = log.NewFromGlobal(log.AddContext("pkg", "grandpa"))
+	ErrUnsupportedSubround = errors.New("unsupported subround")
```

Comment:
nit maybe split that in its own `var` block? It seems strange to me to have it together with the logger.

Or put it in an errors.go file.

Reply:
Done!